.github/workflows/update-gpuci.yml

```diff
@@ -54,31 +58,27 @@ jobs:
       - name: Update new RAPIDS versions
         uses: jacobtomlinson/gha-find-replace@v2
         with:
-          include: 'ci/**'
```

Comment:
Why are these `ci/**` filters no longer necessary? I think *some* filters should be kept, assuming I understand how the templating is working in this repo. Otherwise, you risk modifying things like these `.github/workflows` files, which might lead to surprising results (I don't know, for example, how and when GitHub Actions parses these files).

Reply:
Done in https://github.com/rapidsai/dask-build-environment/pull/105/commits/fec0a76360a86e3e61d53cca0a0d1da80741eb94